diff --git a/superset/utils/date_parser.py b/superset/utils/date_parser.py
index ccbf07da8f..657bd41ab7 100644
--- a/superset/utils/date_parser.py
+++ b/superset/utils/date_parser.py
@@ -120,7 +120,7 @@ def normalize_time_delta(human_readable: str) -> dict[str, int]:
 
     key = matched[2].lower() + "s"
     value = int(matched[1])
-    value = -value if (matched[3] or "").lower() == "ago" else value
+    value = value if (matched[3] or "").lower() == "ago" else -value
     if key == "quarters":
         # pd.DateOffset does not accept a `quarters` argument
         key, value = "months", value * 3
